lib/akamai_rspec/matchers/non_akamai.rb

```diff
@@ -54,12 +54,18 @@ def ssl_client_for_verify_cert(tcp_client, addr, url)
     return false if URI(url).scheme == "http"
     url = "https://#{url}" unless URI(url).scheme
     begin
-      RestClient::Request.execute(method: :get, url: url, verify_ssl: verify)
+      # Avoid AkamaiRspec::Request as it turns off SSL checking
+      @response = RestClient::Request.execute(method: :get, url: url, verify_ssl: verify)
```

Comment:
This doesn't need to be configurable as it's a matcher specifically for checking that SSL verifies correctly.